.ci_support/pyiron_config.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+def main():
+    current_path = os.path.abspath(os.path.curdir)
+    top_level_path = current_path.replace('\\', '/')
```

Comment:
any objections using pathlib instead of os.path? That would avoid manually dealing with the slash.

Reply:
It is now done as you suggested.